File: Baekjoon_Online_Judge/NO_04344/4344.cpp
#include <iostream>
#include <algorithm>
using namespace std;
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 01                             | 
 | 평균은 넘겠지                      | 
 -------------------------------------- 
*/

int main(){
    int C;
    cin >> C;

    for (int i = 0; i < C; i ++){
        int N;
        cin >> N;

        int scores[N], total = 0;
        for (int j = 0; j < N; j ++){
            cin >> scores[j];
            total += scores[j];
        }

        double avg = total / N;
        int over_avg = 0;

        for (auto score: scores){
            if (score > avg) over_avg ++;
        }

        printf("%.3f%%\n", (double)over_avg / N * 100);
    }
    
    return 0;
}

File: Baekjoon_Online_Judge/NO_04344/4344.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 01                             | #
# | 평균은 넘겠지                      | #
# -------------------------------------- #


if __name__ == "__main__":
    C = int(r_input())

    for _ in range(C):
        N, *scores = map(int, r_input().split())
        avg = sum(scores) / N

        over_avg = 0

        for score in scores:
            if score > avg:
                over_avg += 1
        
        print('%.3f' % (over_avg / N * 100) + '%')
